content/soarian/r4/base/management/encounter.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Encounter | R4 API
+---
+
+# Encounter
+
+* TOC
+{:toc}
+
+## Overview
+
+The Encounter resource provides admissions or visits during which health care services were provided to a patient. An encounter has a class to distinguish between different health care settings such as inpatient, outpatient, emergency and so on.
```

Comment:
In the second sentence, please add a comma after "emergency".

Reply:
changes updated.